app/subscriber/src/store/hooks/subscriber/useReports.ts

```diff
@@ -8,10 +8,116 @@ import {
   IReportModel,
   IReportResultModel,
   IUserModel,
+  ReportStatusName,
   sortable,
   useApiSubscriberReports,
 } from 'tno-core';
 
+export interface IReportContentMutationModel {
```

Comment:
Move utility functions and interfaces into separate files.  When everything is in a single file it causes additional merge conflicts down the road.  It's also more difficult to find code when it's buried in a large file.